packages/las-sdk-core/src/client.ts

```diff
@@ -203,30 +203,34 @@ export class Client {
   /**
    * Creates a document, calls the POST /documents endpoint.
    *
-   * @param content Content to POST (base64 string | Buffer)
+   * @param content Content to POST (Buffer)
    * @param contentType MIME type for the document
    * @param options.consentId Id of the consent that marks the owner of the document
    * @param options.groundTruth List of GroundTruth items representing the ground truth values for the document
-   * @param data.description Description of document
-   * @param data.name Name of document
+   * @param options.description Description of document
+   * @param options.name Name of document
    * @returns Document response from REST API
    */
   async createDocument(
-    content: string | Buffer,
+    content: ArrayBuffer | Uint8Array | Buffer,
     contentType: ContentType,
```

Comment:
This argument should be optional now since the fileserver will attempt to read the mime type and update the document.contentType with correct mime type (and consequently we do not need to set contentType explicitly). Any subsequent PUT to the fileserver URL will continue to update the mime type of the document (since document content in practice is mutable now with the fileserver implementation, we also need to make sure the document.contentType reflects changes made to content.)